english/mystory.txt

```diff
@@ -0,0 +1,7 @@
+Once a cat was caught in a hunter's net. A mouse used to live in a nearby hole. The mouse seeing the cat in the net started playing around the cat. 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.